eras/shelley/test-suite/src/Test/Cardano/Ledger/Shelley/Generator/ShelleyEraGen.hs

```diff
@@ -76,11 +76,11 @@ instance
   genEraTxBody _ge _utxo = genTxBody
   genEraAuxiliaryData = genMetadata
 
-  updateEraTxBody _utxo _pp _wits body' fee ins out =
+  updateEraTxBody _utxo _pp _trWits body' fee ins out =
```

Comment:
Wrongful search and replacing function arguments. There are many occurrences of this, could you please revert them back
```suggestion
  updateEraTxBody _utxo _pp _wits body' fee ins out =
```

Reply:
Noted @lehins :memo: :pray: . Will get them reverted.